sparkly/testing.py

```diff
@@ -447,3 +448,101 @@ def setup_data(self):
 
     def teardown_data(self):
         pass
+
+
+class KafkaWatcher():
+    """Context manager that tracks Kafka data published to a topic
+
+    Provides access to the new items that were written to a kafka topic by code running
+    within this context.  This is mainly useful in integration test cases.
```

Comment:
Extra space, actually I'd prefer to move "This is mainly useful in integration test cases" part to "Note:" section to underline this more